src/Fixer/ControlStructure/NoEmptyBlockFixer.php

```diff
@@ -0,0 +1,372 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ControlStructure;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Nat Zimmermann <nat@razorcreations.com>
```

Comment:
Can you change this to `nathanielzimmermann@gmail.com` please? This is an email address for a previous employer that I accidentally used. Or perhaps remove the email entirely if it's not needed.

Reply:
I took it from the commits, so I need to amend author too, will do it later.

Reply:
Thank you!